draft-irtf-cfrg-hpke.md

```diff
@@ -1858,6 +1964,37 @@ contains the following values:
    Each test vector lists the sequence number and corresponding nonce computed
    with `base_nonce`, the plaintext message `pt`, AAD `aad`, and output
    ciphertext `ct`.
+
+{::comment}
+[rfced] FYI, the text rendering of the <tt> element was changed
+in Sept. 2021 (xml2rfc release 3.10.0). <tt> no longer yields quotation
+marks in the text rendering. In light of this, please review and let us
+know if you want to make any changes (e.g., add quotation marks that
+would be in all 3 output formats). For example:
+
+Original:
+       Each test vector lists the sequence number and
+       corresponding nonce computed with "base_nonce", the plaintext
+       message "pt", AAD "aad", and output ciphertext "ct".
+
+Current text:
+       Each test vector lists the sequence
+       number and corresponding nonce computed with base_nonce, the
+       plaintext message pt, AAD aad, and output ciphertext ct.
+
+If you decide to add parentheses similar to usage in Section 9.9:
+       Each test vector lists the sequence
+       number and corresponding nonce computed with base_nonce, the
+       plaintext message (pt), AAD (aad), and output ciphertext (ct).
+
+Side note: <tt> yields fixed-width font in the HTML and PDF files.
+
+This alternative diff file has been provided so that you can review
+changes without the noise of the quotation marks being removed due
+to this change to the rendering of <tt>:
+https://www.rfc-editor.org/authors/rfc9180-alt-diff.html
+{:/comment}
```

Comment:
I'm OK with this rendering in the fixed-width version, as long as the HTML/PDF versions yield fixed-width.  @chris-wood @blipp ?

Reply:
Yep, that works for me.

Reply:
Works for me. Please see my related comment on the definition lists in “Cryptographic Dependencies”.